### DISCUSSION TITLE
Differente of max concurrent and concurrency

### DISCUSSION THREAD

**Discussion by bibiboui** [swar/Swar-Chia-Plot-Manager#293] [Q&A]
hello every body !i have two questions
what is the different between  "max_concurrent"  and "max_concurrent_with_start_early"

and the different of "concurrency_start_early_phase" and "concurrency_start_early_phase_delay"

thanks a lot my friends

**Comment by swar** [comment]
There is documentation on the README about this. This is also not an Issue so please use discussions next time.

**Reply by bibiboui** [reply]
ok sorry

**Reply by swar** [reply]
You're fine I just need to clean up the mess that is Issues right now since theres too much going on there and almost none of it is an issue :)

**Reply by bibiboui** [reply]
it's just that I didn't understand how to use them
